hcipy/coronagraphy/lyot.py

```diff
@@ -12,9 +12,11 @@ class LyotCoronagraph(OpticalElement):
     .. [Soummer2007] Soummer et al. 2007, "Fast computation of Lyot-style
         coronagraph propagation".
 
-    .. notes:: The LyotCoroonagraph tries to automatically get the grid from the focal_plane_mask.
-        However, a bug in AgnosticOpticalElements creates an invalid access which crashes the initialization.
-        The correct grid can be passed explicitely to circumvent this bug by using the focal_plane_mask_grid parameter.
+    Notes
+    -----
+    The LyotCoroonagraph tries to automatically get the grid from the focal_plane_mask.
```

Comment:
Since you're fixing docstring typos... Unless Coroonagraphs are a thing :)

Reply:
Fixed.